pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.black]
+line-length = 78
```

Comment:
If we're gonna take this plunge, maybe it's time to reconsider our line length limit? [Their logic in selecting a default for Black of 88](https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length) seems sound.

Reply:
Sure. That'd just mean we have to configure `flake8` to allow it.

My thought it configuring `black` to limit the line length is that it requires less change-your-code-editor churn for us.

Reply:
Exactly :) Would be nice to get a little line length bonus out of this transition. But even if we don't, this should be set to 79, no?

Reply:
Then I'd have to increase the default size of my vim splits. My cheese is moving too much!

But yes, it should be 79. I had been seeing some flake8 errors that I had thought were related to 79, but they aren't.